src/main/java/org/stratumproject/fabric/tna/behaviour/upf/FabricUpfProgrammable.java

```diff
@@ -298,7 +298,7 @@ public void enablePscEncap() {
                       FABRIC_EGRESS_UPF_GTPU_ENCAP, deviceId);
             return;
         }
-        flowRuleService.applyFlowRules(upfTranslator.buildGtpuWithPscEncapRule(
+        flowRuleService.applyFlowRules(deviceId.hashCode(), upfTranslator.buildGtpuWithPscEncapRule(
```

Comment:
Applies elsewhere, can hashCode be negative? I believe in ONOS we say that key should be non-negative if we want to consider it for in-order processing.